pkg/oci/loader.go

```diff
@@ -180,14 +179,14 @@ func LoadResources(ctx context.Context, s *oci.Storage, imageName *oci.ImageDige
 	tarReader := tar.NewReader(f)
 
 	// TODO: Check hash here?  Or otherwise handle error?
-	resources, err := loadResourcesFromTar(ctx, tarReader)
+	resources, err := loadResourcesFromTar(tarReader)
 	if err != nil {
 		return nil, err
 	}
 	return resources, nil
 }
 
-func loadResourcesFromTar(ctx context.Context, tarReader *tar.Reader) (*repository.PackageResources, error) {
+func loadResourcesFromTar(tarReader *tar.Reader) (*repository.PackageResources, error) {
```

Comment:
Why dropping context here? If anything, context cancelation should be part of the for loop at #195.

Reply:
Slightly lazy one but it shows up as an unused param. There is a few of them in the code which prob should be cleaned up. Unless my understanding of how the context is passed/used in k8s.

Reply:
Unused param